test/mocha/helpers.js

```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+exports.getKeystoreId = function({id}) {
```

Comment:
You might want to call this `localId` -- since the actual keystore ID is the full URL. We've been using `localId` in a number of places and other services to help clear up this potential confusion.